chat.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/ably/ably-go/ably"
+	"github.com/ably/ably-go/ably/proto"
+)
+
+func main() {
+	fmt.Println("Type your clientID")
+	reader := bufio.NewReader(os.Stdin)
+	clientId, _ := reader.ReadString('\n')
```

Comment:
personal preference but make it a command arg. Also might be better to call it a username if this is a chat app.